torch/ao/quantization/fx/qconfig_utils.py

```diff
@@ -237,9 +237,12 @@ def generate_qconfig_map(
 
         elif node.op == "call_method":
             module_path, module_type = node_name_to_scope[node.name]
+            # first use node.target (string) to get the qconfig
+            qconfig = maybe_adjust_qconfig_for_module_type_or_name(
+                qconfig_dict, node.target, module_path, global_qconfig)
```

Comment:
is this related to this PR?  If yes, any chance we can make the comment more descriptive?

Reply:
yeah this is to enable that we can configure "object_type" with call_method strings, this functionality is missing previously and looks like no one has used this before